packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,922 @@
+---
+title: CAPTCHAs
+category: management
+---
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" />
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+# CAPTCHAs API
+
```

Comment:
Can we use the intro from the help doc as an intro? (This wording may change)

> As an option to increase org security, Okta supports CAPTCHA tools to prevent automated sign-in attempts. You can integrate either of two providers: [hCaptcha](https://www.hcaptcha.com/) or [reCAPTCHA v2](https://developers.google.com/recaptcha/docs/invisible). 
> 
> The vendor implementations supported by Okta are both invisible; they each run risk-analysis software in the background during sign in to determine the likelihood that the user is a bot. This risk-analysis is based on the settings that you configure with the provider you choose.
>

Reply:
yeah, I talked with mark, we can link the guide in following page: https://help.okta.com/okta_help.htm?type=oie&id=csh-captcha. They share the same content.